src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("the truck started it`s work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("the truck stopped it`s work");
```

Comment:
Similarly, the stop message should be informative. Consider changing it to "Truck stopped its work".